migrate/20220309224100_create_dao_coin_limit_order_tables.go

```diff
@@ -19,6 +19,7 @@ func init() {
 				scaled_exchange_rate_coins_to_sell_per_coin_to_buy TEXT,
 				quantity_to_fill_in_base_units                     TEXT,
 				operation_type                                     BIGINT,
+				order_type                                         BIGINT,
```

Comment:
doesn't really have to be a big int? smallint is probably most efficient for operation_type and order_type